termo/app/Game.hs

```diff
@@ -1,43 +1,51 @@
 module Game where
 
-import Attempt (showAttemptNum, getAttempt, processAttempt)
+import LetterStatus (LetterStatus(..))
 import Words (getSecretWord)
-import Menu (showScore)
-
+import System.Console.ANSI (setSGR, SGR(SetColor, Reset), ColorIntensity(Vivid), Color(..), ConsoleLayer(Foreground))
+
+statusToColor :: LetterStatus -> Color
+statusToColor NoExist = Red
+statusToColor RightPlace = Green
+statusToColor WrongPlace = Yellow
+statusToColor _ = White 
+
+printStyledLetter :: (Char, LetterStatus) -> IO ()
+printStyledLetter (ch, status) = do
+    setSGR [SetColor Foreground Vivid (statusToColor status)]
+    putStr [ch]
+    setSGR [Reset]
+
+processAndPrintAttempt :: String -> String -> IO ()
+processAndPrintAttempt secretWord attempt = do
+    let statuses = zipWith (compareLetters secretWord) secretWord attempt
+    mapM_ printStyledLetter statuses
+    putStrLn ""
+
+compareLetters :: String -> Char -> Char -> (Char, LetterStatus)
+compareLetters secretWord secretLetter attemptLetter
+    | attemptLetter == secretLetter = (attemptLetter, RightPlace)
+    | attemptLetter `elem` secretWord = (attemptLetter, WrongPlace)
+    | otherwise = (attemptLetter, NoExist)
 
 runGame :: IO ()
 runGame = do
-    -- Variáveis globais do jogo
-    -- TODO: Issue 2
     secretWord <- getSecretWord
     let maxAttempts = 6
-
-    -- Loop do jogo
     loopGame secretWord maxAttempts
 
-
-
 loopGame :: String -> Int -> IO ()
 loopGame secretWord attemptNum = do
-    -- Limpando tela
-    -- putStrLn "\ESC[2J"
-
-    -- Exibindo tentativas restantes na tela
-    showAttemptNum attemptNum
-
-    -- Recebendo input do jogador
-    attempt <- getAttempt
-
-    -- Processando tentativa
-    rightAns <- processAttempt secretWord attempt
+    putStrLn ("Tentativa " ++ show attemptNum ++ ":")
+    attempt <- getLine
+    processAndPrintAttempt secretWord attempt
 
-    -- Verificando se o jogo acabou
-    if rightAns then do
-        showScore secretWord attemptNum
+    if attempt == secretWord then do
```

Comment:
Por que paramos de usar o `processAttempt` pra verificar se é a palavra secreta?

https://github.com/UnBParadigmas2024-2/2024.2_G3_Funcional_Termo/blob/9ee4791b9d32d39b279b0d692c19d0c104e8d2dd/termo/app/Attempt.hs#L40-L46

Reply:
Pois a minha funcao verifica e printa de acordo com o que foi pedido.
compareLetters :: String -> Char -> Char -> (Char, LetterStatus)
compareLetters secretWord secretLetter attemptLetter
    | attemptLetter == secretLetter = (attemptLetter, RightPlace)
    | attemptLetter `elem` secretWord = (attemptLetter, WrongPlace)
    | otherwise = (attemptLetter, NoExist)
    
    Se ele digitar a letra certa no lugar certo será printado de verde
    Se ele digitar a letra certa no lugar errado será printado de amarelo
    Se ele digitar a letra errada será printado de preto.
    
    o Método processAttempt n ta fazendo isso.

Reply:
O problema aqui é que ao fazer isso vc entrou MUITO no escopo da issue #9 (Inclusive o codigo da process atempt ta todo na só nn sendo usado)
Tendo assim duas aboragens pra resolver a mesma coisa. 
Sugestão @Neitan2001 e @fellipepcs tentem entrar em acordo pra umas solucao comum.

Reply:
Vou fazer uma adaptação na função dele, assim ambos contribuem.